homeassistant/components/system_bridge/__init__.py

```diff
@@ -360,5 +359,4 @@ async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
             config_entry.minor_version,
         )
 
-    # User is trying to downgrade from a future version
-    return False
+    return True
```

Comment:
I think we want to fail if the major version is from the future.

Copying Erik's description of how the pattern should be:

1. Fail if the config entries (major) version is from the future.
2. Migrate if the config entries minor version is smaller than the minor version we know about.
3. Do nothing if the config entries minor version is same or larger than the minor version we know about.